dashboard/app/dsl/script_dsl.rb

```diff
@@ -113,8 +130,16 @@ def variants
   end
 
   def endvariants
+    @current_scriptlevel[:properties][:prompt] = @prompt if @prompt
```

Comment:
I figure two screenshots with descriptions and labelled buttons can be pretty self-explanatory, so if a levelbuilder thinks a prompt is unnecessary they can just skip it.